### PR TITLE
OOP-ify Fillz

### PR DESCRIPTION
**Pull request by eddie-c-davis**
## Purpose

This PR converts the `Fillz` test with corresponding `fix_tracer` stencil into a `FillNegativeTracerValues` class.

## Code changes:

- Add `FillNegativeTracerValues` class.
- Add wrapper object to `MapN_Tracer_2d`

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate

### CODE REVIEW

File: fv3core/stencils/fillz.py [hunk 1]
```diff
@@ -99,49 +97,47 @@ def fix_tracer(
...
-):
-    # Same as above, but with multiple tracer fields
-    i1 = spec.grid.is_
-    j1 = spec.grid.js
+class Fillz:
+    """
+    Fix tracers to prevent negative masses.
+    """
+
+    def __init__(self):
```
Comment (FlorianDeconinck) [inline fv3core/stencils/fillz.py:105]: I've been passing the `grid` down in order to squash any reference to `spec`

File: fv3core/stencils/fillz.py [hunk 2]
```diff
@@ -99,49 +97,47 @@ def fix_tracer(
...
-    tracers: Dict[str, Any],
-    im: int,
-    jm: int,
-    km: int,
-    nq: int,
-):
-    # Same as above, but with multiple tracer fields
-    i1 = spec.grid.is_
-    j1 = spec.grid.js
+class Fillz:
```
Comment (FlorianDeconinck) [inline fv3core/stencils/fillz.py:100]: Proposal for class name inspired by the name-list doc: `FillNegativeTracerValues`

**Review by FlorianDeconinck — APPROVED** [review]
Looks good, couple of a proposals

File: fv3core/stencils/fillz.py (hunk 2 above)
Comment (eddie-c-davis) [reply fv3core/stencils/fillz.py]: Implemented, thank you!

**Comment by eddie-c-davis** [comment]
Thank you for the review @FlorianDeconinck !

**Pull request merged**